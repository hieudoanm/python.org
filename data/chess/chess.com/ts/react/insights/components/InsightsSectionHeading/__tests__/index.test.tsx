import { render } from '@testing-library/react';
import { SectionHeading } from '..';

jest.mock('next/navigation', () => ({
  useRouter: jest.fn().mockReturnValue({
    push: jest.fn(),
    asPath: '',
    events: { on: jest.fn(), off: jest.fn() },
  }),
}));

describe('SectionHeading', () => {
  it('to match snapshot', () => {
    const { container } = render(<SectionHeading />);
    expect(container).toMatchSnapshot();
  });
});
