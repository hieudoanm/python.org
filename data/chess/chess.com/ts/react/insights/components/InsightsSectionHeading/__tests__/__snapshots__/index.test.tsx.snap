// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeading to match snapshot 1`] = `
<div>
  <div
    class="text-lg md:text-xl lg:text-2xl"
  />
</div>
`;
